Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -425,6 +439,31 @@ - (void)showAdViewAdForParameters:(id<MAAdapterResponseParameters>)parameters ad
     }
 }
 
+#pragma mark - MANativeAdAdapter Methods
+
+- (void)loadNativeAdForParameters:(id<MAAdapterResponseParameters>)parameters andNotify:(id<MANativeAdAdapterDelegate>)delegate
+{
+    self.placementIdentifier = parameters.thirdPartyAdPlacementIdentifier;
+    [self log: @"Loading Native ad for placement: %@...", self.placementIdentifier];
+    
+    if ( ![[VungleSDK sharedSDK] isInitialized] )
+    {
+        [self log: @"Vungle SDK not successfully initialized: failing Native ad load..."];
+        [delegate didFailToLoadNativeAdWithError: MAAdapterError.notInitialized];
+        return;
+    }
+    self.nativeAdAdapter = [[ALVungleMediationNativeAdAdapter alloc] initVungleNativeAd: delegate];
+    [self.nativeAdAdapter requestNativeAd: self.placementIdentifier];
```

Comment:
Might we need to check if this placement is cached before request a Native Ad?  Thanks a lot. :)

Reply:
this is just an adapter. we cant check at this point.